workflows/workflows_templates/cerbos/policies/workflow_run_entity_input.yaml.j2

```diff
@@ -0,0 +1,27 @@
+# Make changes to the template codegen/templates/cerbos/policies/class_name.yaml.j2 instead.
```

Comment:
Same tiny nit as above, can we update the comment with the path for the override template?

Reply:
Done!